apps/taskman/models.py

```diff
@@ -40,14 +41,30 @@ class JiraResolution(models.TextChoices):
     TEST_PENDING = "Test Pending"
 
 
+WORKFLOW_TO_JIRA_STATUS = {
+    WorkflowModel.OSIMState.NEW: (JiraStatus.NEW, JiraResolution.NONE),
+    WorkflowModel.OSIMState.TRIAGE: (JiraStatus.REFINEMENT, JiraResolution.NONE),
+    WorkflowModel.OSIMState.PRE_SECOND_ASSESSMENT: (
+        JiraStatus.REFINEMENT,
+        JiraResolution.NONE,
+    ),
+    WorkflowModel.OSIMState.SECOND_ASSESSMENT: (
+        JiraStatus.IN_PROGRESS,
+        JiraResolution.NONE,
+    ),
+    WorkflowModel.OSIMState.DONE: (JiraStatus.CLOSED, JiraResolution.DONE),
+    WorkflowModel.OSIMState.REJECTED: (JiraStatus.CLOSED, JiraResolution.WONT_DO),
+}
+
+
 class Task(NullStrFieldsMixin):
     """
     Data cached from Jira tasks
     """
 
     uuid = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
     jira_key = models.CharField(max_length=60)
-    jira_group_key = models.CharField(max_length=60)
+    jira_group_key = models.CharField(max_length=60, blank=True)
```

Comment:
If not to laborious it would be nice at the final stage of preparing the pull request to squash the changes which fixes the things previously introduced in the same pull request.

Reply:
changed to the correct commit it was created.